python_modules/dagster/dagster/_cli/workspace/cli_target.py

```diff
@@ -73,15 +73,43 @@ def are_all_keys_empty(kwargs: Mapping[str, str], keys: Iterable[str]) -> bool:
     "grpc_socket",
 )
 
+import tomli
```

Comment:
nit: any reason for this import to be down here?

Reply:
nope. fixed. good catch.